rllib/algorithms/impala/tests/test_vtrace_v2.py

```diff
@@ -26,15 +28,24 @@ def flatten_batch_and_time_dim(t):
 
 
 class TestVtraceRLModule(unittest.TestCase):
-    def test_vtrace_tf2(self):
-        """Tests V-trace-v2 against ground truth data calculated.
+    """Tests V-trace-v2 against ground truth data calculated.
+
+    Our second implementation of V-Trace (V-trace-v2) is optimized for the least
+    amount of flops. The math in the implementation looks nothing like the math in
```

Comment:
implementation matches the implementation in the original code base.

Reply:
Can you write up the full sentence for me, please? Not sure how do merge this snippet with what is already there.

Reply:
The current implementation of Vtrace in RLlib matches the one written in deepmind's scalable-agent repo.
It is an implementation that is optimized for reducing the number of FLOPs per vtrace calculation via dynamic programming.

Reply:
@avnishn I have taken your suggestion and also made a couple more changes that I think improve the docstrings of our V-Trace classes. Please have a look at the latest commit!

Reply:
gr8